CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ You need to have the following tools installed:
 	- Run `npm install`
 - Start the web server:
 	- Run `npm start`
-- Open `http://localhost:3030`.
+- Open `http://localhost:3030/index`.
```

Comment:
I wouldn't add it here, because it doesn't work on OSX and Linux.

Reply:
I change it to
Open `http://localhost:3030` (or `http://localhost:3030/index` on Windows)